### PR TITLE
Add mplt

### PR DESCRIPTION
**Pull request by jordan-heemskerk**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/mplt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - python test/test_import.py
+
+about:
+  home: "http://github.com/stringhamc/mplt"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: "LICENSE"
+  summary: "Set of niceties wrapping matplotlib for signal processing"
+  doc_url: ""
+  dev_url: ""
```
Comment (marcelotrevisani) [inline recipes/mplt/meta.yaml]: Could you please remove the empty ``doc_url`` and ``dev_url`` ?

File: recipes/mplt/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "2e755db3e8efa9c4dc7728f063458280738d8a4c28e455966c77151173ac791b"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/mplt/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/mplt/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+{% set name = "mplt" %}
+{% set version = "2.0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "2e755db3e8efa9c4dc7728f063458280738d8a4c28e455966c77151173ac791b"
```
Comment (marcelotrevisani) [inline recipes/mplt/meta.yaml]: Could you please remove the quotes from the ``sha256`` ?
```suggestion
  sha256: 2e755db3e8efa9c4dc7728f063458280738d8a4c28e455966c77151173ac791b
```

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**